vllm/model_executor/parallel_utils/cupy_utils.py

```diff
@@ -71,7 +73,7 @@ def init_process_group(world_size: int, rank: int, host: str,
 
     if isinstance(cupy, Exception):
         raise ImportError(
-            "NCCLBackend is not available. Please install cupy.") from cupy
+            "NCCLBackend is not available. Please install cupy==13.0.0.") from cupy
```

Comment:
```suggestion
            "NCCLBackend is not available. Please install cupy-cuda12x==13.0.0.") from cupy
```

Reply:
Thanks for pointing it out. Actually, there are two issues:
1. I changed the PR to use cupy 12.3 instead of 13.0 because cupy 13.0 does not support python 3.8 (I wasn't able to find the wheel in pypi).
2. Users need to install different versions of cupy depending on their env. For example, CUDA 11.8 users should install cupy-cuda11x. ROCm users should install cupy-rocm.